tests/libvmi/cpu.go

```diff
@@ -23,6 +23,12 @@ import (
 	v1 "kubevirt.io/api/core/v1"
 )
 
+func WithCPUCount(cores, threads, sockets uint32) Option {
+	return func(vmi *v1.VirtualMachineInstance) {
+		vmi.Spec.Domain.CPU = &v1.CPU{Cores: cores, Threads: threads, Sockets: sockets}
```

Comment:
Please take a look at the other opts below and make sure to not overwrite `vmi.Spec.Domain.CPU` entirely.

Reply:
Agree. Done.